en/docs/setup/configuring-a-sp-and-idp-using-configuration-files.md

```diff
@@ -2,100 +2,101 @@
 
 This topic provides instructions on how to create a service provider and
 identity provider in the WSO2 Identity Server using configuration files
-which is typically used during the deployment stage. This is done so
+which is typically used during the deployment stage. This is configured so
 that multiple tenants in the Identity Server can have the same identity
 provider.
 
-While this topic need not be specifically about federated
-authentication, this scenario addresses a federated authentication
+This topic does need not be specifically about federated
+authentication. This scenario also addresses a federated authentication
 solution. This scenario requires two WSO2 Identity Server instances,
 where one acts as the external identity provider, and the other acts as
 the service provider. From this point onwards, the Identity Server
 instance that acts as the external identity provider will be referred to
 as **`          identity provider IS         `** and the instance that
 acts as the service provider will be referred to as
-**`          service provider IS         `**. Once we are done with the
-configuration, the **`          service provider IS         `** will
-have the travelocity application configured as a service provider and an
+**`          service provider IS         `**. Once the configurations 
+are finished, the **`          service provider IS         `** will
```

Comment:
the configurations are done,

Reply:
Fixed: 36c85c6dbb7747a3519150db3e2f3302d55bbc54